docs/source/support-bundle/supportbundle.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "SupportBundle"
+---
+
+An OpenAPI Schema for this type is published at: [https://github.com/replicatedhq/kots-lint/blob/main/kubernetes-json-schema/v1.23.6-standalone-strict/supportbundle-troubleshoot-v1beta2.json](https://github.com/replicatedhq/kots-lint/blob/main/kubernetes-json-schema/v1.23.6-standalone-strict/supportbundle-troubleshoot-v1beta2.json).
+
+## SupportBundle Schema
+
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: supportbundle
+spec:
+  collectors: []
+  hostcollectors: []
+  analyzers: []
+  hostanalyzers: []
+  uri: ""
+```
+
+## Properties
+
+### `collectors`
+
+Optional. A list of [`collectors`](https://troubleshoot.sh/docs/collect/).  Returns information collected from the current `kubectl` context.
+
+### `analyzers`
+
+Optional. A list of [`analyzers`](https://troubleshoot.sh/docs/analyze/).  Returns information collected from the current `kubectl` context.
+
+### `hostcollectors`
+
+Optional. A list of [`hostcollector`](https://troubleshoot.sh/docs/host-collect-analyze/overview/) properties.  Returns information from the host where the `support-bundle` or `collect` binary is executed.
+
```

Comment:
given that we're likely to want to deprecate the `collect` binary in the future, shall we leave that off here?

Reply:
we may not deprecate that, let's leave it as is